test_programs/compile_success_with_bug/underconstrained_value_detector_5425/src/main.nr

```diff
@@ -0,0 +1,37 @@
+unconstrained fn maximum_price(options: [u32; 3]) -> u32 {
+    let mut maximum_option = 0;
+    for option in options {
+        if (option > maximum_option) {
+            maximum_option = option;
+        }
+    }
+    maximum_option
+}
+
+fn main(sandwiches: pub [u32; 3], drinks: pub [u32; 3], snacks: pub [u32; 3], best_value: u32) {
+    let meal_deal_cost: u32 = 390;
+    let most_expensive_sandwich = maximum_price(sandwiches);
```

Comment:
Hm, I thought we would have to surround this call with `unsafe` as explained in the [docs](https://github.com/noir-lang/noir/blob/d86891bea9e01d12988d4ee344883d018be32ea7/docs/docs/noir/concepts/unconstrained.md?plain=1#L97-L98) 🤔